### DISCUSSION TITLE
Improve Motion Effects with additional adjustments

### DISCUSSION THREAD

**Discussion by MidwestManedWolf** [elementor/elementor#24563] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When using the motion effects in Elementor Pro we are only able to select preset motions. I would like to see translation options to define from where and how long the animations should be.

### Describe the solution you'd like.

When selecting "fade in from ____" my biggest issue is that the distance is often too long. Every nice fade in from animation has a much shorter distance. So I would like to see a range slider to say from which distance the animation should run in, direction, and duration of the effect. 

### Describe alternatives you've considered.

I have used the scrolling effect animation to capture what I am looking for. Additionally in order to have a short fade in up distance I use a negative margin under the element and then add a spacer below it to keep the right distance from other objects. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
I think additionally if there was a way to add custom animations via some import function would be very useful. Similar to how we can import custom svg icons. Perhaps via Lottie or some other animation type?
